scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -47,7 +47,7 @@ public static BuildTargetResult Publish(BuildTargetContext c)
         [Target(
             nameof(PublishTargets.PublishVersionBadge),
             nameof(PublishTargets.PublishSdkInstallerFile),
-            //nameof(PublishTargets.PublishDebFileToDebianRepo),
+            nameof(PublishTargets.PublishDebFilesToDebianRepo),
```

Comment:
cc @Sridhar-MS do you know why this was commented out? didn't even notice until now.